defaults/main.yml

```diff
@@ -13,7 +13,7 @@ sensu_enterprise_dashboard_package: sensu-enterprise-dashboard
 
 # Sensu repo urls
 sensu_yum_repo_url: "http://repositories.sensuapp.org/yum/$releasever/$basearch/"
-sensu_apt_repo_url: "deb     http://repositories.sensuapp.org/apt {{ ansible_distribution_release }} main"
+sensu_apt_repo_url: "deb     https://repositories.sensuapp.org/apt {{ ansible_distribution_release }} main"
```

Comment:
Is this supposed to be changed to `https://sensu.global.ssl.fastly.net/` instead?